src/MoBi.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -59,6 +74,17 @@ public override async Task<ModelProject> MapToModel(SnapshotProject snapshot, Pr
       return project;
    }
 
+   private void loadProjectContentFromPKSimSnapshot(string snapshot, ModelProject project)
+   {
+      var simulationTransfer = _pkSimStarter.LoadSimulationTransferFromSnapshot(snapshot);
+
+      var simulationConfiguration = simulationTransfer.Simulation.Configuration;
+
+      simulationConfiguration.ModuleConfigurations.Each(x => project.AddModule(x.Module));
+      simulationConfiguration.ExpressionProfiles.Each(project.AddExpressionProfileBuildingBlock);
+      project.AddIndividualBuildingBlock(simulationConfiguration.Individual);
+   }
```

Comment:
observed data is not part of the snapshot?

Reply:
I have to add that in the approved, but still open PK-Sim PR.

Reply:
added